discord/gateway_intents.go

```
@@ -42,6 +42,9 @@ const (
 	GatewayIntentsAll = GatewayIntentsNonPrivileged |
 		GatewayIntentsPrivileged
 
+	GatewayIntentsDefault = GatewayIntentGuilds |
```

I don't think it's good practice to provide defaults for Intents, it's something that should be explicitly set by the user.

---

then default would be `0`
idk if thats better

---

I disagree, imo it should throw an error if not provided

---

that would require it to be a pointer which is ugly >.>

---

then defaulting to 0 is better

---

oki